mappings/net/minecraft/entity/boss/dragon/EnderDragonFight.mapping

```diff
@@ -54,3 +54,4 @@ CLASS net/minecraft/unmapped/C_hxzjzvro net/minecraft/entity/boss/dragon/EnderDr
 		ARG 2 source
 	METHOD m_uzlusufn setSpawnState (Lnet/minecraft/unmapped/C_ubwoswoy;)V
 		ARG 1 spawnState
+	CLASS C_tunbvwoo DragonState
```

Comment:
oh, this is the dragon fight data stuff; I don't think I'd use this name?

Reply:
How about `DragonFightState`?